src/Service/Authenticator/ResourceAccess.php

```diff
@@ -72,11 +73,19 @@ public function authenticate(Request\Request $request)
         }
 
         // check if the given access_token is valid and not expired
-        $accessToken = $this->repository->loadToken(new TokenId($authorization[1]), new TokenType\AccessToken());
+        try {
+            $accessToken = $this->repository->loadToken(new TokenId($authorization[1]), new TokenType\AccessToken());
+        } catch (\Exception $e) {
+            throw new \RuntimeException('Failed to load access token from repository.', 0, $e);
+        }
         if(null === $accessToken) {
```

Comment:
done

Reply:
@SG-Noxoreos hm, I still see issues with the file. E.g. constructor curly braces and equal signs not aligned

Reply:
Sorry, I missed that one. I fixed it after the fact. I guess I need to check my codestyle settings, because it doesn't check for aligned assignments and curly braces.